### PR TITLE
Floofinization

### PR DESCRIPTION
**Pull request by ShatteredSwords**
# Description

Seb became their own thing in the Floof folder, new Seb migrations, and Spawner, NNN was moved to Floof folder. This was cherry-picked from my more items branch, which will have to wait until we get the new Loadouts update from EE.

---

# Changelog

:cl:
- add: Adds Seb themselves as their own thing!
- tweak: Seb will always spawn instead of Siobhan in Floof maps.

### CODE REVIEW

File: Resources/Migrations/migration.yml
Comment (FoxxoTrystan) [inline Resources/Migrations/migration.yml:260]:
```suggestion

# 2024-08-16 Floof Only Please message @FracturedSwords on discord if there are any merge conflicts upcoming
SpawnMobArcticFoxSiobhan: SpawnMobArcticFoxSeb
MobArcticFoxSiobhan: MobArcticFoxSeb
```

We have support to use custom migration folders move it to "floofmigration"

Comment (FoxxoTrystan) [reply Resources/Migrations/migration.yml:260]: now also the one above need also to be moved there, if not ported upstream.

File: Resources/Prototypes/Floof/Entities/Mobs/NPCs/pets.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Mobs/NPCs/pets.yml:1]: You could just parent it off "Siobhan" instead of "MobArcticFox"

File: Resources/Prototypes/Floof/Entities/Objects/Fun/toys.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Objects/Fun/toys.yml:1]: Due to this addtion, this PR will be put on hold until the upstream merge where the "Plushie Update" will bring, this should be modified to fit in the upcoming update.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Comment by FoxxoTrystan** [comment]
Changlog can be smaller then this, please try to compact changlogs.

File: Resources/Prototypes/Floof/Entities/Objects/Fun/toys.yml
Comment (FoxxoTrystan) [inline Resources/Prototypes/Floof/Entities/Objects/Fun/toys.yml:1]: If this file is unused, its should be deleted.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Review by FoxxoTrystan — APPROVED** [review]

**Review by Memeji — APPROVED** [review]

**Pull request merged**
